gradio/blocks.py

```diff
@@ -179,23 +181,30 @@ def __init__(
         variant: column type, 'default' (no background) or 'panel' (gray background color and rounded corners)
         """
         self.variant = variant
-        super().__init__(visible, css)
+        super().__init__(visible=visible, css=css)
 
-    def get_template_context(self):
+    def get_config(self):
         return {
             "type": "column",
             "variant": self.variant,
-            **super().get_template_context(),
+            **super().get_config(),
+        }
+
+    @staticmethod
+    def update(
+        variant: Optional[str] = None,
+        css: Optional[Dict] = None,
+        visible: Optional[bool] = None,
+    ):
+        return {
+            "variant": variant,
```

Comment:
Do we send `None` values to the Frontend, if I remember correctly we were removing them from the dicts. @abidlabs I remember like you had some PR about this in the past.

Reply:
removed them with utils.delete_none